core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/CodeSystem.java

```diff
@@ -55,6 +56,14 @@ public String getResourceType() {
 		return getSettings() == null ? null : (List<String>) getSettings().get(CommonSettings.LOCALES);
 	}
 	
+	
+	public static ResourceURI uriFromPath(String resourcePath) {
```

Comment:
Please eliminate this method. Services should never ever try to calculate a `ResourceURI` from a branch path expression.

Reply:
Done